### PR TITLE
Now filters.group supports callback queries

### PR DESCRIPTION
**Pull request by Princic-1837592**

### CODE REVIEW

File: pyrogram/filters.py [hunk 1]
```diff
@@ -448,7 +448,8 @@ async def private_filter(_, __, m: Message):
 # endregion
 
 # region group_filter
-async def group_filter(_, __, m: Message):
+async def group_filter(_, __, u: Update):
+    m = u.message if isinstance(u, CallbackQuery) else u
```
Comment (AndrielFR) [inline pyrogram/filters.py:452]: But what if the callback is a `via_bot` one?

Comment (Princic-1837592) [reply pyrogram/filters.py:452]: The filter fails, as it did when applied to a callback query

Comment (AndrielFR) [reply pyrogram/filters.py:452]: sad

**Comment by delivrance** [comment]
We should extend the same enhancement to all relevant filters (private, channel, others?) and use an approach similar to `filters.regex` (multiple cases and raising an error for non-applicable updates). Plus, the `message` attribute for callback queries can be None; this needs to be handled.

**Pull request closed without merging**